cleverhans/attacks_eager.py

```diff
@@ -0,0 +1,331 @@
+from abc import ABCMeta
+import numpy as np
+from six.moves import xrange
+import warnings
+import collections
+import tensorflow as tf
+from distutils.version import LooseVersion
+import cleverhans.utils as utils
+
+_logger = utils.create_logger("cleverhans.attacks_eager")
+
+if LooseVersion(tf.__version__) < LooseVersion('1.8.0'):
+    error = ('For eager execution',
+             'use Tensorflow version greather than 1.8.0.')
+    print(error)
+    exit(1)
+
+try:
+    from cleverhans.model_eager import Model_Eager
+except ImportError:
+    error = 'Import Error: cleverhans.model_eager'
+    raise ImportError(error)
```

Comment:
Just for my understanding, why do you intercept the import error if you're reraising it? Is it only to modify the error message? For what reason?

Reply:
tensorflow version (require >= 1.8.0) is to checked before importing Model_Eager, 
but PEP8 requires all import statements to be at beginning of the file.

i used try ... except as a hack around this deadlock, if there is an elegant solution please let me know.

Reply:
Shouldn't it raise an error if tf version is too old and eager is not present?
What's wrong with just:
```
from cleverhans.model_eager import Model_Eager
```
If it's present, it gets imported and if it's not present it will raise an import error.

Reply:
True, above code is redundant now, will modify it in updated PR.